x-pack/plugins/cases/server/features/v1.ts

```diff
@@ -97,6 +123,7 @@ export const getCasesKibanaFeature = (): KibanaFeatureConfig => {
                   delete: [APP_ID],
                 },
                 ui: capabilities.delete,
+                replacedBy: [{ feature: FEATURE_ID_V2, privileges: ['cases_delete'] }],
```

Comment:
Can we use const here (same for settings)?

```suggestion
                replacedBy: [{ feature: FEATURE_ID_V2, privileges: [CASES_DELETE_SUB_PRIVILEGE_ID] }],
```